### PR TITLE
PR: toPython helpers for QDate and QTime

### PR DESCRIPTION
**Pull request by bob-schumaker**
This adds toPython() compat functions to QDate and QTime for symmetry.

### CODE REVIEW

File: qtpy/QtCore.py [hunk 1]
```diff
@@ -21,7 +21,9 @@
 
     # For issue #153 and updated for issue #305
     from PyQt5.QtCore import QDateTime
```
Comment (CAM-Gerlach) [inline qtpy/QtCore.py:23]:
```suggestion
    from PyQt5.QtCore import QDate, QDateTime, QTime
```

Shouldn't they be explicitly imported here too?

**Review by CAM-Gerlach — COMMENTED** [review]
Seems naively reasonable to me overall, but I'd defer to @dalthviz on that.

Could you either add tests like the [one for `QDateTime.toPython`](https://github.com/spyder-ide/qtpy/blob/e21b92ff9a645d05311f7512aeb14c1fdd7f5697/qtpy/tests/test_qtcore.py#L25) (or parameterize that test accordingly)?

**Review by dalthviz — COMMENTED** [review]
Hi @bob-schumaker thank you for working on this! I would say that the missing thing here is to add some tests for `QDate.toPython` and `QTime.toPython` as @CAM-Gerlach mentioned. If you need help with that let us know!

Other than that this LGTM :+1:

**Comment by bob-schumaker** [comment]
> Hi @bob-schumaker thank you for working on this! I would say that the missing thing here is to add some tests for `QDate.toPython` and `QTime.toPython` as @CAM-Gerlach mentioned. If you need help with that let us know!
> 
> Other than that this LGTM 👍

Duh, out of practice. Let me add the tests :-)

**Review by dalthviz — APPROVED** [review]
Thanks @bob-schumaker !

**Pull request merged**